tests/API/SEAP/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+set(LIBRARIES openscap ${LIBXML2_LIBRARIES} ${LIBXSLT_LIBRARIES} ${LIBXSLT_EXSLT_LIBRARIES} ${PCRE_LIBRARIES} ${CURL_LIBRARIES} ${RPM_LIBRARIES})
+add_executable(test_api_seap_concurency test_api_seap_concurency.c)
+target_link_libraries(test_api_seap_concurency ${LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+add_executable(test_api_seap_list test_api_seap_list.c)
+target_link_libraries(test_api_seap_list ${LIBRARIES})
+add_executable(test_api_seap_number test_api_seap_number.c)
+target_link_libraries(test_api_seap_number ${LIBRARIES})
+add_executable(test_api_seap_spb test_api_seap_spb.c)
+target_include_directories(test_api_seap_spb PUBLIC ${CMAKE_SOURCE_DIR}/src/OVAL/probes/SEAP/generic)
+target_link_libraries(test_api_seap_spb ${LIBRARIES})
+add_executable(test_api_seap_string test_api_seap_string.c)
+target_link_libraries(test_api_seap_string ${LIBRARIES})
+add_executable(test_api_seap_parser test_api_seap_parser.c)
+target_link_libraries(test_api_seap_parser ${LIBRARIES})
+add_executable(test_api_sexp_ID test_api_sexp_ID.c)
+target_link_libraries(test_api_sexp_ID ${LIBRARIES})
+add_executable(test_api_SEXP_deepcmp test_api_SEXP_deepcmp.c)
+target_link_libraries(test_api_SEXP_deepcmp ${LIBRARIES})
+add_executable(test_api_strto test_api_strto.c)
+target_include_directories(test_api_strto PUBLIC ${CMAKE_SOURCE_DIR}/src/OVAL/probes/SEAP/generic)
+target_link_libraries(test_api_strto ${LIBRARIES})
```

Comment:
use functions, this is copy paste hell. same with other test files. It all links to the same things and uses pretty much the same cmake features, make it a function.

Reply:
@mpreisler I will add a function